azure-iot-device/tests/iothub/conftest.py

```diff
@@ -8,7 +8,7 @@
 import pytest
 
 # These fixtures are shared between sync and async clients
-from .client_fixtures import auth_provider, pipeline
+from .client_fixtures import auth_provider, pipeline, pipeline_manual_cb
```

Comment:
in conftest.py, you can use pytest_plugins to import fixtures into your test: pytest_plugins = ["tests.iothub.client_fixtures", "tests.common.pipeline_test_fixtures"].  This way you don't have to list every fixture in an import statement.

Reply:
I think this is a good idea, but it triggers a warning when I attempt to do it right now, so let's punt this down the road.

---
In reply to: [290983779](https://github.com/Azure/azure-iot-sdk-python-preview/pull/101#discussion_r290983779) [](ancestors = 290983779)